### PR TITLE
Final-Adding batch account with keyvault pfx password

### PR DESCRIPTION
**Pull request by manishshar**
All passed pull request.

### CODE REVIEW

File: 101-batch-with-keyvault-pfx-password/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ARM template for using Azure Batch service with pfx password certificate from keyvault
```
Comment (bmoore-msft) [inline 101-batch-with-keyvault-pfx-password/README.md]: Shorten the title

Azure Batch Service with Certificate from KeyVault

File: 101-batch-with-keyvault-pfx-password/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "batchservice": {
+      "value": "GET-PREREQ-batchservice"
+    },
+    "certdata": {
+     "value": "GEN-SELFSIGNED-CERT-PFXDATA"
+    },
+    "certthumbprint": {
+      "value": "5384a8be6df0f82717d08e18028636aac668b6a7"
```
Comment (bmoore-msft) [inline 101-batch-with-keyvault-pfx-password/azuredeploy.parameters.json]: use the GEN value for the thumbprint here

File: 101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "objectId": {
+      "type": "string",
+      "defaultValue": "9337ef34-9e90-41f7-XXXX-c2175660fa80"
```
Comment (bmoore-msft) [inline 101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json]: remove this

**Comment by manishshar** [comment]
@bmoore-msft : Thanks for response. Do we have this available--> GEN-SELFSIGNED-CERT-THUMBPRINT 
because GEN-SELFSIGNED-CERT-PFXDATA does not having thumbprint GEN-SF-CERT-THUMBPRINT. In short, PFX is not generated for SF thumbprint.

SEE Error generated:
The specified data and the thumbprint do not match. Thumbprint from cert data is 5384a8be6df0f82717d08e18028636aac668b6a7 and specified is 59bc5cbf0e7410815922346ddc21b83ce53ed80e

File: 101-batch-with-keyvault-pfx-password/README.md (hunk 1 above)
Comment (manishshar) [reply 101-batch-with-keyvault-pfx-password/README.md]: Done

File: 101-batch-with-keyvault-pfx-password/azuredeploy.parameters.json (hunk 2 above)
Comment (manishshar) [reply 101-batch-with-keyvault-pfx-password/azuredeploy.parameters.json]: Done

File: 101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json (hunk 3 above)
Comment (manishshar) [reply 101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json]: Done

**Comment by manishshar** [comment]
@bmoore-msft : Thanks for helping. Could you please check and let me know if anything else is required to merge in Azure.com index.

**Comment by manishshar** [comment]
@bmoore-msft : Not sure if you are receiving my messages. Could you please check and let me know if anything else is required to merge in Azure index

Best Regards
Manish

**Pull request closed without merging**